module/zfs/arc.c

```diff
@@ -465,6 +465,35 @@ static uint_t zfs_arc_lotsfree_percent = 10;
  */
 static int zfs_arc_prune_task_threads = 1;
 
+/*
+ * Controls the number of ARC eviction threads returned by num_evict_threads()
+ * Possible values:
+ * 0  (disabled) parallel eviction threads are not used.
+ * 1  (auto) compute the number of threads using a logarithmic formula.
+ * 2+ (manual) set the number manually, limited by zfs_arc_evict_threads_max.
+ */
+static uint_t zfs_arc_evict_threads = 1;
+
+/*
+ * The mumber of allocated ARC eviction threads. This limits the maximum value of
+ * zfs_arc_evict_threads.
+ * The number is set up at module load time and depends on the initial value of
+ * zfs_arc_evict_threads. If zfs_arc_evict_threads is set to auto, a logarithmic
+ * function is used to compute this value. Otherwise, it is set to max_ncpus.
+ */
+static uint_t zfs_arc_evict_threads_max;
+
+/*
+ * Compute the number of ARC eviction threads to use.
+ */
+static inline uint_t
+num_evict_threads(void)
```

Comment:
You do not need to specify inline for small static function used only once. Compiler will do it by itself.  But honestly I don't think this needs to be a function for two lines.  It only complicates code reading.

Reply:
Removed the function.